nextflow.config

```diff
@@ -22,7 +22,7 @@ params {
     genome_reference = 'grch37'
 
     // Options: Predictions
-    tools = 'mhcflurry'
+    tools = 'mhcnuggets'
```

Comment:
Isnt that a change of the default prediction tool? Seems a pretty impactful change, maybe that could be mentioned in the CHANGELOG & documentation? (Well, just saw its aiming at 3.x instead of 2.x, so that might cover it partially)